custom_components/bhyve/switch.py

```diff
@@ -145,7 +155,6 @@ async def async_setup_platform(hass, config, async_add_entities, _discovery_info
         program_device = device_by_id.get(program.get("device_id"))
         program_id = program.get("program")
         if program_id is not None:
-            _LOGGER.info("Creating switch: Program %s", program.get("name"))
```

Comment:
Is this deliberate?

Reply:
That was when I was debugging the uuid issue, I shifted it to the `__init__` of `BHyveProgramSwitch()` to match the others.

Reply:
That was actually from when I was debugging the uuid issue, and understanding how everything hangs together. But no real reason.
I added it to `__init__` in `BHyveProgramSwitch()` to match logging for the other classes.